cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreTableManager.java

```diff
@@ -461,7 +468,13 @@ private String generateHivePartitionKey(PartitionKey key) {
       String fieldName = entry.getKey();
       Comparable fieldValue = entry.getValue();
       String quote = fieldValue instanceof String ? "'" : "";
-      builder.append(sep).append(fieldName).append("=").append(quote).append(fieldValue.toString()).append(quote);
+      builder.append(sep);
+      if (shouldEscapeColumns) {
+        builder.append('`').append(fieldName).append('`');
```

Comment:
I can just check for it. According to the docs, a literal backtick is just two backticks.